docs/docs/tutorial.md

```diff
@@ -1,36 +1,26 @@
 ---
-title: Mesos-DNS on GCP
+title: Mesos-DNS Set Up
 ---
 
-## Mesos-DNS on Google Compute Platform
+## Mesos-DNS on a Mesos Cluster
 
-This is a step-by-step tutorial for running Mesos-DNS with [Mesosphere](https://google.mesosphere.com/) on [Google Compute Platform](https://cloud.google.com). 
+This is a step-by-step tutorial for running Mesos-DNS on a Mesos cluster.
 
-### Step 1: Launch a Mesosphere cluster
+### Step 1: Launch a Mesos cluster
 
-Launch a development Mesosphere cluster following the instructions at [https://google.mesosphere.com/](https://google.mesosphere.com/). Note that you will need a [Google Cloud Platform](https://cloud.google.com) account with an active project that has billing enabled.
+Follow the instructions on the Mesosphere Community Documentation site to [set up a Mesos and Marathon cluster](https://open.mesosphere.com/getting-started/install/).
 
-The development cluster includes 1 master and 3 slaves.  We will use the information on the example cluster listed below for this tutorial. Note that we will use the internal IP addresses for all nodes in all cases. The cluster runs Mesos (version 0.21.1) and Marathon (version 0.7.6).
-
-<p class="text-center">
-  <img src="{{ site.baseurl}}/img/example-cluster-gce.png" width="650"  alt="">
-</p>
-
-[Google Cloud Platform](https://cloud.google.com) blocks port `53` by default. To unblock traffic for port `53` follow these [directions](http://stackoverflow.com/questions/21065922/how-to-open-a-specific-port-such-as-9090-in-google-compute-engine). Note that you need to unblock port `53` for both `tcp` and `udp` traffic. At the end, if you examine the firewall rules for your cluster you have a rule such as the one listed below, which opens port `53` for `udp` and `tcp` traffic from hosts in the `10.0.0.0` subnet. 
-
-<p class="text-center">
-  <img src="{{ site.baseurl}}/img/example-firewall-gce.png" width="350"  alt="">
-</p>
+Have the IP addresses of your cluster to hand, since we'll be installing and running Mesos DNS by sshing into your nodes. Make sure that port `53` is unblocked for both `tcp` and `udp` traffic between your nodes. To unblock traffic for port `53` on Google Compute Engine, follow these [directions](http://stackoverflow.com/questions/21065922/how-to-open-a-specific-port-such-as-9090-in-google-compute-engine).
 
 ### Step 2: Build and install Mesos-DNS
```

Comment:
We should probably change this section to avoid building Mesos-DNS since we ship stable binaries -- people should just use those instead of building their own.